test/SerialIOTest.cpp

```diff
@@ -4139,7 +4179,9 @@ void variableBasedSeries( std::string const & file )
 
 TEST_CASE( "variableBasedSeries", "[serial][adios2]" )
 {
+#if openPMD_HAVE_ADIOS2
```

Comment:
You could also mask out the whole test by moving this around the `TEST_CASE` a whole - then it does not show up in the list of available tests if ADIOS2 is not compiled in.

Reply:
updated :)